examples/triton/setup.sh

```diff
@@ -97,7 +97,26 @@ check() {
         exit 1
     fi
 
-    echo CONSUL=consul.svc.${TRITON_ACCOUNT}.${TRITON_DC}.cns.joyent.com > _env
+
+    echo '# Prometheus discovery for Triton' > _env
+    echo TRITON_ACCOUNT=${TRITON_ACCOUNT} >> _env
+    echo TRITON_DC=${TRITON_DC} >> _env
```

Comment:
I was intentionally trying avoid DC-specific details in the `_env` file, that's why this was commented out in https://github.com/autopilotpattern/prometheus/pull/18/files#diff-86aea33913dc22bc9ba97b5f7b8c956eR102